### PR TITLE
Feature/add orderitems to updateorderdto

### PR DESCRIPTION
**Pull request by NullableInt**

### CODE REVIEW

File: src/SwedbankPay.Sdk.Infrastructure/TransactionDto.cs [hunk 1]
```diff
@@ -34,6 +34,12 @@ internal ITransaction Map()
             var state = string.IsNullOrEmpty(State) ? "Unknown" : State;
             var id = new Uri(Id, UriKind.RelativeOrAbsolute);
 
+            IProblem problem = null;
+            if(Problem != null)
+            {
+                problem = Problem.Map();
+            }
```
Comment (asbjornu) [inline src/SwedbankPay.Sdk.Infrastructure/TransactionDto.cs:41]: Can't you just use `null` propagation?

```suggestion
            var problem = Problem?.Map();
```

Comment (NullableInt) [reply src/SwedbankPay.Sdk.Infrastructure/TransactionDto.cs]: I can!
I love `null` propagation but I keep forgetting it exists.

File: src/SwedbankPay.Sdk.Infrastructure/PaymentInstruments/TransactionResponse.cs [hunk 2]
```diff
@@ -7,7 +7,10 @@ internal class TransactionResponse : Identifiable, ITransactionResponse
         public TransactionResponse(Uri id, TransactionDto transaction)
             : base(id)
         {
-            Transaction = transaction.Map();
+            if(transaction != null)
+            {
+                Transaction = transaction.Map();
+            }
```
Comment (asbjornu) [inline src/SwedbankPay.Sdk.Infrastructure/PaymentInstruments/TransactionResponse.cs:13]: `null` propagation FTW?

```suggestion
            Transaction = transaction?.Map();
```

File: src/SwedbankPay.Sdk.Infrastructure/PaymentOrders/PaymentOrderUpdateRequestDetailsDto.cs [hunk 3]
```diff
@@ -1,18 +1,26 @@
...
+	{
+		public PaymentOrderUpdateRequestDetailsDto(PaymentOrderUpdateRequestDetails paymentOrder)
+		{
+			Amount = paymentOrder.Amount.InLowestMonetaryUnit;
+			Operation = paymentOrder.Operation.Value;
+			VatAmount = paymentOrder.VatAmount?.InLowestMonetaryUnit;
+            foreach (var item in paymentOrder.OrderItems)
+            {
+				OrderItems.Add(new OrderItemDto(item));
+            }
```
Comment (asbjornu) [inline src/SwedbankPay.Sdk.Infrastructure/PaymentOrders/PaymentOrderUpdateRequestDetailsDto.cs:15]: What's going on with indentation here?

**Review by asbjornu — CHANGES_REQUESTED** [review]

**Review by asbjornu — APPROVED** [review]

**Pull request merged**